lib/features/modeling/behavior/LayoutConnectionBehavior.js

```diff
@@ -0,0 +1,117 @@
+import {
+  assign
+} from 'min-dash';
+
+import inherits from 'inherits';
+
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+import { getConnectionAdjustment as getConnectionAnchorPoint } from './util/ConnectionLayoutUtil';
+
+/**
+ * A component that makes sure that Associations connected to Connections
+ * are updated together with the Connection.
+ *
+ * @param {EventBus} eventBus
+ * @param {Modeling} modeling
+ */
+export default function LayoutConnectionBehavior(
```

Comment:
Codecov reports some missing test coverage for this one. Is there anything we can do against it?

Reply:
The Missing tests are for outgoing connections. Currently, text annotations always use incoming connections. I'm not sure if there is a use-case for outgoing connections, but it feels weird to only have logic for one and not the other

Reply:
I guess the bpmn spec allows associations between 2 sequence flows, but we don't support those yet

Reply:
> Currently, text annotations always use incoming connections

Disregard that, it works, we just can't model them. I'll add tests